### ISSUE TITLE
Alpine x-init called multiples times

### ISSUE THREAD

**Issue by Mushr0000m** [livewire/livewire#2520]
Having AlpineJS components encapsulated that are using `x-init` make all nested components `x-init` called each time livewire re-render the HTML.

Here is a very simple example : [https://laravelplayground.com/#/snippets/0c06fa8a-d6c9-4d37-8631-726ff60cdfb9](https://laravelplayground.com/#/snippets/0c06fa8a-d6c9-4d37-8631-726ff60cdfb9). Each time you change something in the input it call the second component `x-init` and log `bar` in the console.

**Context**
- Livewire version: 2.3.13
- Laravel version: 8
- Alpine version: 2.8.0
- Browser: Chrome / Firefox

This is a big issue for our needs, I read all the topics about this kind of issue but can't find a fix or if there will be a fix for that ? In our case we can't just use `wire:ignore` we just need to have `x-init` being called once...

**Comment by joshhanley** [comment]
@Mushr0000m Have a look at PR #2434. It fixes an issue with doubling up of Alpine click handles which we believe should also fix the x-init running multiple times issue. But it's waiting on a merge in Alpine first. 

I believe we should close this and keep this discussion on the original x-init double up issue (even though it's labelled with entangle) as I believe it's the same issue. 

Hope this helps!

**Comment by Mushr0000m** [comment]
Thanks I hope it will be approved quickly ;-)

**Comment by austenc** [comment]
Still working on it, but we will get this fixed soon! Closing since this is a duplicate of #763

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)
